include/unifex/let_value.hpp

```diff
@@ -56,17 +56,18 @@ struct _successor_receiver<Operation, Values...>::type {
 
   template <typename... SuccessorValues>
   void set_value(SuccessorValues&&... values) && noexcept {
+    auto& op = op_;
```

Comment:
You also need to apply the same change to the `set_done()` and `set_error()` methods.
Otherwise, looks good.